src/ui/components/Timeline/Capsule.tsx

```diff
@@ -0,0 +1,82 @@
+import classNames from "classnames";
+import { CircularProgressbar, buildStyles } from "react-circular-progressbar";
+import { useSelector } from "react-redux";
+import { getIndexingProgress, getLoadingStatusWarning, LoadingStatusWarning } from "ui/actions/app";
+
+import Icon from "../shared/Icon";
+
+import styles from "./Capsule.module.css";
+import { EditFocusButton } from "./EditFocusButton";
+import FocusInputs from "./FocusInputs";
+
+export default function Capsule() {
+  const indexingProgress = useSelector(getIndexingProgress) || 0;
```

Comment:
Is this the right selector to be using?

I think I might want to switch to a selector that calculates 0-100% differently. Rather than the union of index and loaded, I think it might make more sense to say 100% is 100% of both of those (that way we wouldn't show no updates for a while even when e.g. indexing is updating).

Reply:
I wrote a new one. Feels better.